app/models/katello/content_view.rb

```diff
@@ -152,6 +151,18 @@ def as_json(options = {})
       result
     end
 
+    def total_package_count(env)
+      repoids = self.repos(env).collect { |r| r.pulp_id }
+      result = Katello::Package.legacy_search('*', 0, 1, repoids)
+      result.length > 0 ? result.total : 0
+    end
+
+    def total_puppet_module_count(env)
+      repoids = self.repos(env).collect { |r| r.pulp_id }
+      result = Katello::PuppetModule.legacy_search('*', :page_size => 1, :repoids => repoids)
+      result.length > 0 ? result.total : 0
+    end
```

Comment:
Forgot I restored a file locally and then didn't remove it again -- https://github.com/Katello/katello/pull/5188/files#diff-c8ca002e08c4fd6cbd537ca3122e9e38L46